src/components/UI/SearchBar/index.css

```diff
@@ -13,12 +14,32 @@
   outline: none;
   background: var(--background);
   width: 80%;
+  padding-right: 2.5em;
 }
 
-.SearchBar > label {
-  color: var(--secondary);
+.clearSearchInputWrapper {
+  position: absolute;
+  top: 22%;
```

Comment:
a common trick to absolute position this in the center ( so you don't have to use values like this 22%) is to do

```
top: 50%
transform: translateY(-50%)
```

Reply:
Thanks! That's super useful to know. Should be pushed soon